scripts/parse-pip-requirements-from-environment-yaml.py

```diff
@@ -29,7 +29,9 @@
             pattern=r'([^=])=([^=])',
             repl='\\1==\\2',
             string=item)
-        new_dependencies.append(new_item)
+        if "conda" not in new_item and not new_item.startswith("python"):
```

Comment:
Was that not handled with the `excluded_dependencies` below?  Oh, I see, it's now no longer an exact match.  Maybe move it down there just to consolidate?

Honestly this whole ~script~ hack can probably just go away if we decide that we're consolidating on some form of conda, but let's leave that for a future PR.

Reply:
do you want me to consolidate or leave it for moving to conda?

Reply:
My suggestion for now is to just switch to something like this below for now:

```
-    if item in excluded_dependencies:
+    dep_name = re.split('[><=]+', item)[0]
+    if dep_name in excluded_dependencies:
```

(tested locally, seems to work as expected)